components/security/crowdsec.mdx

```diff
@@ -1,11 +1,205 @@
 ---
-title: Crowdsec
-description: Curated Threat Intelligence Powered by the Crowd - now at the edge
+title: CrowdSec
+description: Crowd-sourced threat intelligence and IP blocking at the edge - protect your website from malicious actors in real-time
 ---
 
+<img src="/images/banners/crowdsec.png" className="rounded-lg" noZoom />
+
 import EdgeeSdk from '/snippets/edgee-sdk.mdx';
-import WaitingHouston from '/snippets/waiting/houston.mdx';
 
 <EdgeeSdk />
-<WaitingHouston />
+
+[CrowdSec](https://www.crowdsec.net/) is an open-source security engine that leverages crowd-sourced threat intelligence to identify and block malicious IP addresses. The CrowdSec component integrates with Edgee to provide real-time IP filtering at the edge using CrowdSec's Raw IP List integration, automatically blocking known threats while allowing legitimate traffic through.
+
+This component fetches regularly updated blocklists containing IP addresses flagged by the global CrowdSec community for malicious behavior such as brute force attacks, scanning activities, and other threat patterns.
+
+## Getting Started
+
+**To integrate CrowdSec into your Edgee project:**
+
+<Steps>
+<Step title="Create CrowdSec Console account">
+  1. Sign up for a free [CrowdSec Console account](https://app.crowdsec.net/)
+  2. Verify your email and complete the account setup
+</Step>
+
+<Step title="Set up Raw IP List integration">
+  1. Log into your CrowdSec Console
+  2. Navigate to **Blocklists > Integrations**
+  3. Find "Raw IP List" and click **Connect**
+  4. Enter a unique name for your integration (e.g., "Edgee Integration")
+  5. Click **Generate** to create your credentials
```

Comment:
Looking at the Raw IP List documentation: https://docs.crowdsec.net/u/integrations/rawiplist

It seems like at this step you also get access to the Endpoint URL. I would clarify it here, so you can mention it below (instead of the very long URL).

Reply:
Adding this image might be useful too: https://docs.crowdsec.net/img/console_integrations_generic_credentials_dark.png

Reply:
> I would clarify it here, so you can mention it below (instead of the very long URL).

I think i will keep the URL template while mentioning the earlier step, so that the user can be sure it's the correct URL by comparing the formats